### ISSUE TITLE
Add install wheel(s) for CUDA 11.7

### ISSUE THREAD

**Issue by BotScutters** [facebookresearch/pytorch3d#1434]
## 🚀 Feature
In particular, right now I'm in a bit of a dependency trap where I need PyTorch3D built for CUDA 11.7 (ideally for Python 3.10, torch 1.13.1), but it seems like there are only pre-built wheels up to 11.6.

### Bonus Feature Request
Would it be possible to have a page somewhere listing which actual pre-built wheels are available for installation through pip? The install.md page is kind of non-specific about it and generally out-of-date.

## Motivation
I know that it's been stated that this is a conda-first library, and that generally the recommendation around here is to just use conda to install pytorch3d. I just keep finding myself on projects where there's a strong buy-in to pip and reluctance to add in Conda to the build and process if it's not absolutely necessary. The line around here about pip seems to center around only supporting it for the sake of Colab, but I can't be the only one out here that depends on the pip wheels for use in non-colab settings. Additional Linux installation wheels on PyPi would be tremendously useful to me and I'm sure many others.

I've found myself in this position several times over the last year whenever I have to go update a container or project--every library I want to install is straightforward _except_ for pytorch3d, where I inevitably find myself in a guess-and-check loop trying to figure out what wheels might be available and how I can resolve my dependency chain, or if I'm going to need to build from source for some reason. This time, while trying to figure out what combination of Python and Torch I could use with a Cuda 11.7 build, I just ended up writing this little script to just do a grid search check:
```
pys=(38 39 310)
cus=(113 114 115 116 117)
pyts=(1120 1121 1130 1131)

for py in "${pys[@]}"; do
  for cu in "${cus[@]}"; do
    for pyt in "${pyts[@]}"; do
      url="https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py${py}_cu${cu}_pyt${pyt}/download.html"
      status_code=$(curl -L -s -o /dev/null -w "%{http_code}" $url)
      if [ "$status_code" == "200" ]; then
        echo "Wheel available for Python $py, CUDA $cu, PyTorch $pyt: $url"
      fi
    done
  done
done
```

This told me that there just aren't any pre-built wheels for Cuda 11.7. Clearly this kind of search shouldn't really be necessary, which is why I added my bonus request, but I decided to post it here for the benefit of others like me hunting for this information. This is the output I got, btw:
```
Wheel available for Python 38, CUDA 113, PyTorch 1120: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py38_cu113_pyt1120/download.html
Wheel available for Python 38, CUDA 113, PyTorch 1121: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py38_cu113_pyt1121/download.html
Wheel available for Python 38, CUDA 116, PyTorch 1130: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py38_cu116_pyt1130/download.html
Wheel available for Python 38, CUDA 116, PyTorch 1131: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py38_cu116_pyt1131/download.html
Wheel available for Python 39, CUDA 113, PyTorch 1120: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1120/download.html
Wheel available for Python 39, CUDA 113, PyTorch 1121: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1121/download.html
Wheel available for Python 39, CUDA 116, PyTorch 1130: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu116_pyt1130/download.html
Wheel available for Python 39, CUDA 116, PyTorch 1131: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu116_pyt1131/download.html
Wheel available for Python 310, CUDA 113, PyTorch 1120: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py310_cu113_pyt1120/download.html
Wheel available for Python 310, CUDA 113, PyTorch 1121: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py310_cu113_pyt1121/download.html
Wheel available for Python 310, CUDA 116, PyTorch 1130: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py310_cu116_pyt1130/download.html
Wheel available for Python 310, CUDA 116, PyTorch 1131: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py310_cu116_pyt1131/download.html
```

**Comment by BotScutters** [comment]
To clarify my resistance to using Conda for the installation--my application runs inside of a ROS2 environment, which is, in general, [not compatible with Conda](https://docs.ros.org/en/foxy/How-To-Guides/Using-Python-Packages.html#using-python-packages-with-ros-2).

**Comment by bottler** [comment]
Done - py310_cu117_pyt1131 is now present.

**Comment by BotScutters** [comment]
Thank you for the quick help!

**Comment by madvn** [comment]
@bottler could you also please make `py310_cu116_pyt1121` available?

**Comment by aradhyamathur** [comment]
are wheels available for torch 2.1.0 as well ?

**Comment by chenerg** [comment]
Hope for a full list about supported pytorch versions, cuda versions and python versions.

**Comment by nitinmukesh** [comment]
Yes please add wheel for all CUDA and python versions (maybe starting 3.9 and above). Different AI tools using pytorch3d have different requirement.
It will be big help for non technical people like me.

**Comment by bottler** [comment]
@chenerg @nitinmukesh Conda packages are the recommended binaries, and exist for many combinations of dependencies. They are harder to install incorrectly. The wheels really only exist for use with google colab, where conda seems to be a pain to use. We are not planning to add many more wheels.

**Comment by BotScutters** [comment]
@bottler just FYI, there are some environments where conda simply isn't an option, such as on robotic systems which use ROS--that's the case with my team, where we depend on the availability of the pip wheels you host. So I understand that your team's stance is that they "only exist for use with google colab", but realistically there are lots of users out there (I assume more than just me) that rely on these wheels. Whenever we need to update version, we've actually resorted to writing scripts that brute-force search your wheel repository to try to figure out which wheels actually exist.
And all of that said, we've chatted before and you've always been tremendously helpful and responsive when I've reached out, so thank you very much for your active help and support!
